services/sensu/service.go

```diff
@@ -133,9 +136,10 @@ func (s *Service) prepareData(name, output string, level alert.Level) (*net.TCPA
 
 	postData := make(map[string]interface{})
 	postData["name"] = name
-	postData["source"] = c.Source
+	postData["source"] = hostname
 	postData["output"] = output
 	postData["status"] = status
+	postData["auto_resolve"] = false
```

Comment:
In that case we should create a new property on the SensuHandler called noAutoResolve(), as I think most users will want the auto resolve behavior.

For reference auto_resolve docs https://sensuapp.org/docs/0.23/reference/checks.html